README.md

```diff
@@ -324,6 +324,14 @@ Your NDK is typically located within your SDK:`<sdk>/ndk/<ndk-version>`
 
 Our prebuilt libraries target Android NDK 25.1.8937393.
 
+Android build uses gradle, which requires a JVM installed.
+
+For example, on a windows machine, open a terminal within the nakama-cpp\android folder and run:
```

Comment:
there is a text above which don't seem relevant anymore as graddle should be handling NDK versions